content/ja/docs/reference/kubectl/generated/kubectl.md

```diff
@@ -0,0 +1,336 @@
+<td colspan="2">--as string</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>この操作を実行する際に偽装するユーザー名を指定します。ユーザーには、通常のユーザーまたは名前空間内のサービスアカウントを指定できます。</p></td>
+</tr>
+
+<tr>
+<td colspan="2">--as-group strings</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>この操作で偽装するグループを指定します。複数のグループを指定する場合は、このフラグを繰り返し使用できます。</p></td>
+</tr>
+
+<tr>
+<td colspan="2">--as-uid string</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>この操作で偽装するUIDを指定します。</p></td>
+</tr>
+
+<tr>
+<td colspan="2">--cache-dir string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: "$HOME/.kube/cache"</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>デフォルトのキャッシュディレクトリ</p></td>
+</tr>
+
+<tr>
+<td colspan="2">--certificate-authority string</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>証明書の検証に使用する証明機関の証明書ファイルへのパス</p></td>
+</tr>
+
+<tr>
+<td colspan="2">--client-certificate string</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>TLSで使用するクライアント証明書ファイルへのパス</p></td>
+</tr>
+
+<tr>
+<td colspan="2">--client-key string</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>TLSで使用するクライアントキーのファイルへのパス</p></td>
+</tr>
+
+<tr>
+<td colspan="2">--cluster string</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>使用するkubeconfigのクラスタ名</p></td>
+</tr>
+
+<tr>
+<td colspan="2">--context string</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>使用するkubeconfigのコンテキスト名</p></td>
+</tr>
+
+<tr>
+<td colspan="2">--default-not-ready-toleration-seconds int&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: 300</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>notReady:NoExecuteのtolerationに対するtolerationSecondsを指定します。該当のtolerationを持たないPodにはデフォルトでこの値が追加されます。</p></td>
+</tr>
+
+<tr>
+<td colspan="2">--default-unreachable-toleration-seconds int&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: 300</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>unreachable:NoExecuteのtolerationに対するtolerationSecondsを指定します。該当のtolerationを持たないPodにはデフォルトでこの値が追加されます。</p></td>
+</tr>
+
+<tr>
+<td colspan="2">--disable-compression</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>trueにすると、サーバーへのすべてのリクエストに対するレスポンス圧縮を無効にします</p></td>
+</tr>
+
+<tr>
+<td colspan="2">-h, --help</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>kubectlのヘルプを表示します</p></td>
+</tr>
+
+<tr>
+<td colspan="2">--insecure-skip-tls-verify</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>trueにすると、サーバー証明書の有効性を検証しません。この設定はHTTPS接続を非安全にします</p></td>
+</tr>
+
+<tr>
+<td colspan="2">--kubeconfig string</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>CLIリクエストで使用するkubeconfigファイルのパス</p></td>
+</tr>
+
```

Comment:
```suggestion

<tr>
<td colspan="2">--kuberc string</td>
</tr>
<tr>
<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>設定に使用するkubercファイルへのパス。この機能は、KUBECTL_KUBERC=false、またはKUBERC=offの環境変数を設定し、エクスポートすることで無効化できます。</p></td>
</tr>

```
`--kuberc string`フラグが追加されていました。
> Path to the kuberc file to use for preferences. This can be disabled by exporting KUBECTL_KUBERC=false feature gate or turning off the feature KUBERC=off.

KUBECTL_KUBERCをフィーチャーゲートとするのは違和感があったため、KUBERCとともに環境変数として訳しました。